src/Summoner/Question.hs

```diff
@@ -45,12 +45,13 @@ printQuestion question (def:rest) = do
     putTextLn $ "/" <> restSlash
 printQuestion question [] = T.putStrLn question
 
-choose :: Text -> [Text] -> IO Text
+choose :: (Show a, IsString a) => Text -> [a] -> IO a
```

Comment:
I guess here `IsString` won't work for enums. 
The solution that I can imagine is to make `Read a` constraint instead and then you can do `readmaybe @a answer` to decide what to do next. How do you think?

Reply:
I don`t mind. :)
> I guess here IsString won't work for enums.

Is it by reason of enum instance lack?